test/binaryen.js/kitchen-sink.js

```diff
@@ -1165,15 +1171,24 @@ function test_expression_info() {
   module.dispose();
 }
 
-test_types();
-test_features();
-test_ids();
-test_core();
-test_relooper();
-test_binaries();
-test_interpret();
-test_nonvalid();
-test_parsing();
-test_internals();
-test_for_each();
-test_expression_info();
+function test(m) {
+  try {
+    m();
+  } catch(e) {
+    console.trace();
```

Comment:
How helpful is this? The stack trace of the error is on `e` anyhow, so I'm not sure what this can add here in this catch block, but I'm not familiar with `console.trace`...

Reply:
in theory console.trace prints the stack trace of the last raised error, see https://developer.mozilla.org/fr/docs/Web/API/console/trace_static
but it doesn't seem to help much tbh, so I can remove it

Reply:
Done